post.md

```diff
@@ -0,0 +1,702 @@
+---
+post_title: The art of disassembly
```

Comment:
@BartekLew maybe the title should be "AT&T assembly introduction for firmware developers" or other more descriptive title - we have to take care of SEO

Reply:
@pietrushnic Yes, I think it'd be better to sparate AT&T assembly basics and Executable format overview and exctend a little bit original topic and simply add links.